src/style.css

```diff
@@ -1 +1,48 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+  justify-content: flex-start;
+}
+
+.stars__star {
+  font-family: Arial, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. It's a good practice to specify a generic family as a fallback. For example, `font-family: Arial, 'Helvetica Neue', Helvetica, sans-serif;` ensures that if Arial is not available, the browser will try Helvetica Neue, then Helvetica, and finally any available sans-serif font.